app/dashboard/layout.tsx

```
@@ -0,0 +1,143 @@
+          sidebarOpen ? "translate-x-0" : "-translate-x-full"
+        }`}
+      >
+        <div className="flex h-16 items-center justify-between border-b border-zinc-200 px-4 dark:border-zinc-800">
+          <Link href="/dashboard" className="flex items-center space-x-2">
+            <FileSpreadsheet className="h-8 w-8 text-blue-600" />
+            <span className="text-xl font-bold">BelegBoost</span>
+          </Link>
+          <button
+            onClick={() => setSidebarOpen(false)}
+            className="rounded-md p-2 hover:bg-zinc-100 dark:hover:bg-zinc-800 lg:hidden"
+          >
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)